src/modules/commander/PreflightCheck.cpp

```diff
@@ -576,6 +576,19 @@ static bool ekf2Check(orb_advert_t *mavlink_log_pub, vehicle_status_s &vehicle_s
 
 	// check accelerometer delta velocity bias estimates
 	param_get(param_find("COM_ARM_EKF_AB"), &test_limit);
+	for (uint8_t index=13; index<16; index++) {
+		// allow for higher uncertainty in estimates for axes that are less observable to prevent false positives
+		// adjust test threshold by 3-sigma
+		float test_uncertainty = 3.0f * sqrtf(fmaxf(status.covariances[index],0.0f));
+		if (fabsf(status.states[index]) > test_limit + test_uncertainty) {
+			if (report_fail) {
+				mavlink_log_critical(mavlink_log_pub, "Preflight Fail: High Accelerometer Bias");
+			}
+
+			success = false;
+			goto out;
+		}
+	}
 
 	if (fabsf(status.states[13]) > test_limit || fabsf(status.states[14]) > test_limit
```

Comment:
Remove the original "High Accelerometer Bias" check?

Reply:
Reporting the reason for failure if report_fail is true is consistent with the way other test failures are handled.

Reply:
Isn't this new "High Accelerometer Bias" check with uncertainty a replacement for the existing "High Accelerometer Bias"? The existing check is still going to trigger, and hence not reduce the number of false positives.

Reply:
Will fix.